src/table.c

```diff
@@ -167,3 +167,1009 @@ const uint16_t table_n_b[] = {
     0b111101111101,  // 7th messiaen mode
     0b100110011001,  // augmented
 };
+
+// preset psuedo random bit mask drum notes for TRE
+const char table_t_r_e[216][3] = { { 0, 0, 0 },
```

Comment:
might be worth adding a comment here that if the number of presets is increased, `drum_helpers.h` boundary checks will need to be updated. or could add a constant for the pattern count and use it here.